### PR TITLE
[Bug-Fix] Modify queries to exclude ineligible ppts from chaser email

### PR DESCRIPTION
**Pull request by tonyheadford**
### Context

In a recent chaser email to schools it was noticed in support that some participants did not have QTS. This fixes the queries so that they do not include participants that have other eligibility issues.

### Changes proposed in this pull request
Check the `ECFParticipantEligibility` record for other flags and only include those that have no induction

### CODE REVIEW

File: app/queries/ects/with_an_appropriate_body_and_unregistered_query.rb [hunk 1]
```diff
@@ -6,8 +6,9 @@ def call
       InductionRecord
         .active
         .training_status_active
-        .joins(:participant_profile)
+        .joins(participant_profile: :ecf_participant_eligibility)
         .where(participant_profile: { induction_start_date: nil, type: "ParticipantProfile::ECT" })
+        .where(ecf_participant_eligibility: { qts: true, active_flags: false, different_trn: false, previous_induction: false, no_induction: true })
```
Comment (peteryates) [inline app/queries/ects/with_an_appropriate_body_and_unregistered_query.rb:11]: Do these conditions apply anywhere else too? Worth combining them into a scope?

**Review by peteryates — APPROVED** [review]
Looks good, left a open ended question

File: app/queries/ects/with_an_appropriate_body_and_unregistered_query.rb (hunk 1 above)
Comment (tonyheadford) [reply app/queries/ects/with_an_appropriate_body_and_unregistered_query.rb:11]: Good question. It might make it clearer if these were in a scope/scopes.

**Review by peteryates — APPROVED** [review]
Nice!

**Pull request merged**